mlflow/pyfunc/__init__.py

```diff
@@ -627,30 +628,34 @@ def _load_model_or_server(model_uri: str, env_manager: str):
         local_path,
         env_manager=env_manager,
         install_mlflow=os.environ.get("MLFLOW_HOME") is not None,
-        create_env_root_dir=True,
+        create_env_root_dir=False,
```

Comment:
I found `mlflow.evaluate(..., env_manager="virtualenv")` always installs Python. Not sure if this behavior is intended, but this is a temporary change for quick iterrations.

Reply:
Are you saying that it always installs Python even if the Python version has already been installed before?

Reply:
Yes. I don't think this is intended.

Reply:
We can confirm this by:

```
> git diff
diff --git a/examples/evaluation/evaluate_on_binary_classifier.py b/examples/evaluation/evaluate_on_binary_classifier.py
index 73d064e10..ab0b588a3 100644
--- a/examples/evaluation/evaluate_on_binary_classifier.py
+++ b/examples/evaluation/evaluate_on_binary_classifier.py
@@ -29,6 +29,7 @@ with mlflow.start_run() as run:
         targets="label",
         model_type="classifier",
         evaluators=["default"],
+        env_manager="virtualenv",
     )

> python examples/evaluation/evaluate_on_binary_classifier.py
> python examples/evaluation/evaluate_on_binary_classifier.py
```